client/my-sites/upgrades/domain-management/controller.jsx

```diff
@@ -319,6 +319,11 @@ module.exports = {
 	},
 
 	domainManagementTransferToOtherUser( pageContext ) {
+		const selectedSite = sites.getSelectedSite();
+		if ( selectedSite && selectedSite.options.is_automated_transfer ) {
+			page.redirect( '/domains/manage' + ( selectedSite ? ( '/' + selectedSite.slug ) : '' ) );
```

Comment:
There shouldn't be a need for the ternary here. You're checking for truthyness of `selectedSite` in the conditional above. So you can just use a template literal for the URL:
```js
page.redirect( `/domains/manage/{ selectedSite.slug }` );
```
Also, I think you want an early return here, so we don't run through the rest of the function unnecessarily. Redirection isn't going to stop the rest of the code from running.